# HG changeset patch
# User Rik <rik@octave.org>
# Date 1471383871 25200
#      Tue Aug 16 14:44:31 2016 -0700
# Node ID c563396c5bf1660394c272b9954d832bd11d55f2
# Parent  1ddb53b6ad303d840e03b4b84b3562706b18a3e3
annotation.m: Fix %!demo #3 to work with dump_plots.m

annotation.m: Don't try and place headstyle in double quotes.

diff --git a/scripts/plot/appearance/annotation.m b/scripts/plot/appearance/annotation.m
--- a/scripts/plot/appearance/annotation.m
+++ b/scripts/plot/appearance/annotation.m
@@ -1351,22 +1351,22 @@ endfunction
 %! ns = numel (styles);
 %! nrows = ceil (ns/2);
 %! dy = 1/nrows;
 %! y = 1 - dy/2;
 %!
 %! jj = 1;
 %! for ii = 1:nrows
 %!   annotation ("textarrow", [0.3 0.5], [y y], ...
-%!               "string", ['"' styles{jj} '"'], "fontsize", 15, ...
+%!               "string", styles{jj}, "fontsize", 15, ...
 %!               "headstyle", styles{jj}, "textcolor", "b");
 %!   jj = jj + 1;
 %!   if (jj <= ns)
 %!     annotation ("textarrow", [0.7 0.5], [y y], ...
-%!                 "string", ['"' styles{jj} '"'], "fontsize", 15, ...
+%!                 "string", styles{jj}, "fontsize", 15, ...
 %!                 "headstyle", styles{jj}, "textcolor", "b");
 %!   jj = jj + 1;
 %!   endif
 %!   y = y - dy;
 %! endfor
 %! annotation ("line", [0.5 0.5], [dy/2 1-dy/2], "linestyle", "-.");
 
 %!demo
